[
  {
    "number": 1017,
    "title": "FCOIN \u662f\u4e0d\u662f\u9650\u5236\u4e0b\u5355\u554a\uff0c\u62115\u79d2\u4e0b\u4e00\u6b21\u5355\uff0c\u4e0b\u4e8620\u6b21\u5de6\u53f3\u5c31\u4e0b\u4e0d\u4e86\u4e86\u3002",
    "created_at": "2018-08-03T11:48:49Z",
    "closed_at": "2018-08-04T13:50:27Z",
    "labels": [
      "question"
    ],
    "url": "https://github.com/vnpy/vnpy/issues/1017",
    "body": "## \u73af\u5883\r\n\r\n* \u64cd\u4f5c\u7cfb\u7edf: \u5982Windows 7\u6216\u8005Ubuntu 16.04\r\n* Anaconda\u7248\u672c: \u5982Anaconda 4.0.0 Python 2.7 32\u4f4d\r\n* vn.py\u7248\u672c: \u5982v1.7\u53d1\u884c\u7248\u6216\u8005dev branch 20170920\uff08\u4e0b\u8f7d\u65e5\u671f\uff09\r\n\r\n## Issue\u7c7b\u578b\r\n\u4e09\u9009\u4e00\uff1aBug/Enhancement/Question\r\n\r\n## \u9884\u671f\u7a0b\u5e8f\u884c\u4e3a\r\n\r\n\r\n## \u5b9e\u9645\u7a0b\u5e8f\u884c\u4e3a\r\n\r\n\r\n## \u91cd\u73b0\u6b65\u9aa4\r\n\r\n\u9488\u5bf9Bug\u7c7b\u578bIssue\uff0c\u8bf7\u63d0\u4f9b\u5177\u4f53\u91cd\u73b0\u6b65\u9aa4\u4ee5\u53ca\u62a5\u9519\u622a\u56fe\r\n\r\n",
    "comments_url": "https://api.github.com/repos/vnpy/vnpy/issues/1017/comments",
    "author": "sage-peng",
    "comments": [
      {
        "user": "vnpy",
        "created_at": "2018-08-04T13:50:27Z",
        "body": "FCOIN\u7684\u6d41\u63a7\u975e\u5e38\u4e25\uff0c\u5b98\u7f51\u53f7\u79f010\u79d2100\u6b21\uff0c\u5b9e\u9645\u53ef\u80fd10\u79d2\u53ea\u67092-3\u6b21\uff0c\u800c\u4e14\u9891\u7e41\u8d85\u8fc7\u4f1a\u88ab\u7981IP"
      },
      {
        "user": "sage-peng",
        "created_at": "2018-08-05T10:18:26Z",
        "body": "\u90a3\u8fd9\u4e2a\u95ee\u9898\u53ef\u4ee5\u5173\u95ed\u4e86"
      }
    ],
    "satisfaction_conditions": [
      "Explanation of FCOIN's API rate limiting policies",
      "Clarification of order placement thresholds and timing constraints",
      "Identification of consequences for exceeding API limits"
    ],
    "_classification": {
      "category": "Requires build environment but hard to be dockerized",
      "timestamp": "2025-04-04 23:50:19"
    }
  },
  {
    "number": 2409,
    "title": "\u6253\u5f00vnstation\uff0c\u6216\u8005\u662f\u547d\u4ee4\u884c\u8f93\u5165python run.py\uff0c\u90fd\u4f1a\u51fa\u73b0\u5982\u4e0b\u4fe1\u606f\uff1aThis Windows version (6.3.9600) does not support the required bluetooth API.",
    "created_at": "2020-04-05T08:26:29Z",
    "closed_at": "2020-04-13T14:40:00Z",
    "labels": [
      "help wanted"
    ],
    "url": "https://github.com/vnpy/vnpy/issues/2409",
    "body": "## \u73af\u5883\r\n\r\n* \u64cd\u4f5c\u7cfb\u7edf: \u5982Windows 8\u7248\u672c\uff086.3.9600\uff09\r\n* Python\u7248\u672c: 3.7\r\n* vn.py\u7248\u672c: 2.1.1\r\n\r\n## Issue\u7c7b\u578b\r\n\u4e09\u9009\u4e00\uff1aBug/Enhancement/Question\r\n\r\n## \u9884\u671f\u7a0b\u5e8f\u884c\u4e3a\r\n\r\n\r\n## \u5b9e\u9645\u7a0b\u5e8f\u884c\u4e3a\r\n\r\n\r\n## \u91cd\u73b0\u6b65\u9aa4\r\n\r\n\u9488\u5bf9Bug\u7c7b\u578bIssue\uff0c\u8bf7\u63d0\u4f9b\u5177\u4f53\u91cd\u73b0\u6b65\u9aa4\u4ee5\u53ca\u62a5\u9519\u622a\u56fe\r\n\r\n",
    "comments_url": "https://api.github.com/repos/vnpy/vnpy/issues/2409/comments",
    "author": "moyuweiqing",
    "comments": [
      {
        "user": "moyuweiqing",
        "created_at": "2020-04-07T08:39:12Z",
        "body": "\u76ee\u524d\u5df2\u7ecf\u89e3\u51b3\uff0c\u53ef\u80fd\u662fwin8\u7cfb\u7edf\u4e0b\u4e0d\u652f\u6301\u5427\uff0c\u4f7f\u7528win10\u7cfb\u7edf\u53ef\u4ee5\u6b63\u5e38\u8fd0\u884c"
      },
      {
        "user": "vnpy",
        "created_at": "2020-04-13T14:40:00Z",
        "body": "\u5230\u6587\u4ef6\u5939\u4e2d vnstudio\u4e2d\\Lib\\site-packages\\PyQt5\\Qt\\bin\\\r\n\r\n\u5c06Qt5Bluetooth.dll \u91cd\u547d\u540d\u4e3a Qt5Bluetooth.dll1"
      }
    ],
    "satisfaction_conditions": [
      "Addresses Bluetooth API compatibility with Windows 8 (6.3.9600)",
      "Provides a workaround that doesn't require OS upgrade to Windows 10",
      "Resolves program startup failure related to Bluetooth dependencies",
      "Identifies components with OS version dependencies"
    ],
    "_classification": {
      "category": "Requires build environment but hard to be dockerized",
      "timestamp": "2025-04-04 23:51:30"
    }
  },
  {
    "number": 2362,
    "title": "OKEX\u4ea4\u6613\u6240loadbar \u5728\u4ea4\u6613\u6240\u6570\u636e\u6709\u9650\u7684\u60c5\u51b5\uff0c\u53ea\u80fd\u88c5\u8f7d\u90e8\u5206\u6570\u636e\uff0c\u9020\u6210\u5b9e\u76d8\u4ea4\u6613\u8fdf\u8fdf\u4e0d\u80fd\u542f\u52a8",
    "created_at": "2020-02-25T11:32:28Z",
    "closed_at": "2020-03-10T13:21:38Z",
    "labels": [
      "help wanted"
    ],
    "url": "https://github.com/vnpy/vnpy/issues/2362",
    "body": "## \u73af\u5883\r\n\r\n* \u64cd\u4f5c\u7cfb\u7edf: \u5982Windows 10\r\n\r\n* vn.py\u7248\u672c: v2.10\r\n\r\n## Issue\u7c7b\u578b\r\n\u4e09\u9009\u4e00\uff1aEnhancement\r\n\r\n## \u9884\u671f\u7a0b\u5e8f\u884c\u4e3a\r\n\u521d\u59cb\u5316\u4e4b\u540e\uff0c\u4e0b\u8f7d\u7684\u6570\u636e\u8db3\u591f\u591a\uff0c\u53ef\u4ee5\u9a6c\u4e0a\u4ea4\u6613\r\n\r\n## \u5b9e\u9645\u7a0b\u5e8f\u884c\u4e3a\r\n\u521d\u59cb\u5316\u4e4b\u540e\uff0c\u4e0b\u8f7d\u7684\u6570\u636e\u4e0d\u8db3\uff0c\u4e0d\u80fd\u9a6c\u4e0a\u4ea4\u6613\r\n\r\n## \u91cd\u73b0\u6b65\u9aa4\r\n\r\n\u9488\u5bf9Bug\u7c7b\u578bIssue\uff0c\u8bf7\u63d0\u4f9b\u5177\u4f53\u91cd\u73b0\u6b65\u9aa4\u4ee5\u53ca\u62a5\u9519\u622a\u56fe\r\n\u5728OKEX\u4ea4\u6613\u7684\u65f6\u5019\uff0c\u4fee\u6539query_history\u51fd\u6570\uff0c\u6539\u4e3a\u4eceonetoken\u4e0b\u8f7d\u5386\u53f2\u6570\u636e",
    "comments_url": "https://api.github.com/repos/vnpy/vnpy/issues/2362/comments",
    "author": "wangdalei001",
    "comments": [
      {
        "user": "vnpy",
        "created_at": "2020-03-10T13:21:38Z",
        "body": "\u5df2\u6709\u89e3\u51b3\u65b9\u6848\uff1a\r\n1. \u8bf7\u4f7f\u7528DataRecorder\u5f55\u5236\u6570\u636e\u540e\r\n2. \u5728\u7b56\u7565\u4e2d\u8c03\u7528load_bar\u51fd\u6570\u65f6\uff0c\u4f20\u5165\u53ef\u9009\u53c2\u6570use_database=True\uff0c\u5f3a\u5236\u4f7f\u7528\u672c\u5730\u6570\u636e\u5e93\u7684\u6570\u636e"
      }
    ],
    "satisfaction_conditions": [
      "Ensures sufficient historical data availability during strategy initialization",
      "Provides persistent local data storage integration"
    ],
    "_classification": {
      "category": "Requires build environment but hard to be dockerized",
      "timestamp": "2025-04-04 23:51:42"
    }
  },
  {
    "number": 1508,
    "title": "server 2008\u542f\u52a8vntrader \u8fde\u63a5bitmex\u7684REAL\u548cTESTNTE\u6709\u95ee\u9898\uff0cwin7\u5c31\u6ca1\u6709\u95ee\u9898",
    "created_at": "2019-03-21T00:21:06Z",
    "closed_at": "2019-03-25T03:46:32Z",
    "labels": [
      "help wanted"
    ],
    "url": "https://github.com/vnpy/vnpy/issues/1508",
    "body": "## \u73af\u5883\r\n\r\n* \u64cd\u4f5c\u7cfb\u7edf: \u5982Server 2008 64 \u548c windows 7 64\r\n* Anaconda\u7248\u672c: \u5982Anaconda 18.12 Python 3.7.2 64\u4f4d\r\n* vn.py\u7248\u672c: \u5982v2.0\u53d1\u884c\u7248\u6216\u8005dev branch 20190101\uff08\u4e0b\u8f7d\u65e5\u671f\uff09\r\n\r\n## Issue\u7c7b\u578b\r\n\u4e09\u9009\u4e00\uff1aBug/Enhancement/Question\r\n\r\n## \u9884\u671f\u7a0b\u5e8f\u884c\u4e3a\r\nserver\u548cwin7\u90fd\u4e00\u6837\u6b63\u5e38\u8fde\u63a5\r\n\r\n## \u5b9e\u9645\u7a0b\u5e8f\u884c\u4e3a\r\nserver\u4e0d\u6b63\u5e38\uff1a\u8fde\u63a5bitmex\uff08ctp\u8fde\u63a5\u6b63\u5e38\uff09\r\n   REST API\u542f\u52a8\u6210\u529f\r\n   \u89e6\u53d1\u5f02\u5e38\uff0c\u72b6\u6001\u7801\uff1a<class 'websocket._exceptions.WebSocketProxyException'>\uff0c\u4fe1\u606f\uff1a[WinError 10054] \u8fdc\u7a0b\u4e3b\u673a\u5f3a\u8feb\u5173\u95ed\u4e86\u4e00\u4e2a\u73b0\u6709\u7684\u8fde\u63a5\u3002\r\n connect_bitmex.json\u4e2d proxy_host \u548c proxy_port \u7528\u7684\u662f\u9999\u6e2f\u670d\u52a1\u5668IP\u548c\u7aef\u53e3\r\nwin7\u6b63\u5e38:\r\n  REST API\u542f\u52a8\u6210\u529f\r\n  Websocket API\u8fde\u63a5\u6210\u529f\r\n   Websocket API\u9a8c\u8bc1\u6388\u6743\u6210\u529f\r\n  connect_bitmex.json\u4e2d proxy_host \u548c proxy_port \u7528\u7684\u662f    \r\n  \"proxy_host\": \"127.0.0.1\",\r\n   \"proxy_port\": 1080\r\n\r\n\r\n## \u91cd\u73b0\u6b65\u9aa4\r\n\r\n\u9488\u5bf9Bug\u7c7b\u578bIssue\uff0c\u8bf7\u63d0\u4f9b\u5177\u4f53\u91cd\u73b0\u6b65\u9aa4\u4ee5\u53ca\u62a5\u9519\u622a\u56fe\r\n\r\n",
    "comments_url": "https://api.github.com/repos/vnpy/vnpy/issues/1508/comments",
    "author": "followdao",
    "comments": [
      {
        "user": "vnpy",
        "created_at": "2019-03-21T01:10:34Z",
        "body": "\u7528\u7684\u662f2.0\u7248\u672c\u8fd8\u662f1.9.2\u7248\u672c\u7684\uff1f\r\n\r\n\u5982\u679c\u662f2.0\u7248\u672c\uff0c\u8d70\u4ee3\u7406\u670d\u52a1\u5668\u9700\u8981\u5b89\u88c5shadowsocks"
      },
      {
        "user": "followdao",
        "created_at": "2019-03-21T01:28:58Z",
        "body": "\u9648\u8001\u5e08\uff0c\u6211\u8fd9\u8fb9\u662f2.0.1-DEV\u7248\u672c\uff0c\u5728\u9999\u6e2f\u4e91\u670d\u52a1\u5668\u8fd0\u884c\uff0c\u4e5f\u9700\u8981\u518d\u5b89\u88c5shadowsocks\u5417\uff1f"
      },
      {
        "user": "vnpy",
        "created_at": "2019-03-21T14:48:03Z",
        "body": "\u4f60\u7684\u60c5\u51b5\uff0cproxy_host\u548cproxy_port\u90fd\u76f4\u63a5\u7559\u7a7a\uff0c\u4e0d\u8981\u8f93\u5165\u4efb\u4f55\u4e1c\u897f\u5c31\u884c"
      },
      {
        "user": "followdao",
        "created_at": "2019-03-23T01:55:30Z",
        "body": "\u975e\u5e38\u611f\u8c22\u9648\u8001\u5e08,biemex\u6b63\u5e38\u8fde\u63a5\u4e86"
      }
    ],
    "satisfaction_conditions": [
      "Resolve proxy configuration conflicts specific to Windows Server 2008 environments",
      "Ensure compatibility with server environment network restrictions",
      "Maintain functional equivalence between development (Win7) and production (Server 2008) environments",
      "Provide clear guidance on proxy requirement dependencies"
    ],
    "_classification": {
      "category": "Requires build environment but hard to be dockerized",
      "timestamp": "2025-04-04 23:51:57"
    }
  },
  {
    "number": 653,
    "title": "ImportError: No module named vnxtpquote for win7 32",
    "created_at": "2017-12-21T15:36:54Z",
    "closed_at": "2017-12-22T13:39:17Z",
    "labels": [
      "help wanted"
    ],
    "url": "https://github.com/vnpy/vnpy/issues/653",
    "body": "## \u73af\u5883\r\n\r\n* \u64cd\u4f5c\u7cfb\u7edf: Windows 7 32bit\r\n* Anaconda\u7248\u672c: \u5982Anaconda2 5.0.1 Python 2.7 32\u4f4d\r\n* vn.py\u7248\u672c: dev branch 20171221\uff08\u4e0b\u8f7d\u65e5\u671f\uff09\r\n\r\n## Issue\u7c7b\u578b\r\n\u4e09\u9009\u4e00\uff1aQuestion\r\n\r\n## \u9884\u671f\u7a0b\u5e8f\u884c\u4e3a\r\n\u6253\u5f00\u793a\u4f8b\u7a97\u53e3\r\n\r\n## \u5b9e\u9645\u7a0b\u5e8f\u884c\u4e3a\r\n\u547d\u4ee4\u884c\u62a5\u9519\r\n\r\n## \u91cd\u73b0\u6b65\u9aa4\r\n\u6b63\u5e38\u7684\u5b89\u88c5\u4e86\u5168\u5957\u5de5\u5177\uff0c\u5728\u6700\u540e\u6d4b\u8bd5\u7684\u65f6\u5019\u62a5\u9519\r\n`\r\nY:\\py\\vnpy\\examples\\VnTrader>python run.py\r\nC:\\ProgramData\\Anaconda2\\lib\\site-packages\\vnpy\\api\\ib\\__init__.py:3: RuntimeWar\r\nning: to-Python converter for class std::vector<class ibapi::shared_ptr<class Or\r\nderCondition>,class std::allocator<class ibapi::shared_ptr<class OrderCondition>\r\n > > already registered; second conversion method ignored.\r\n  from vnib import *\r\nTraceback (most recent call last):\r\n  File \"run.py\", line 19, in <module>\r\n    from vnpy.trader.gateway import (ctpGateway, oandaGateway,\r\n  File \"C:\\ProgramData\\Anaconda2\\lib\\site-packages\\vnpy\\trader\\gateway\\xtpGatewa\r\ny\\__init__.py\", line 4, in <module>\r\n    from xtpGateway import XtpGateway\r\n  File \"C:\\ProgramData\\Anaconda2\\lib\\site-packages\\vnpy\\trader\\gateway\\xtpGatewa\r\ny\\xtpGateway.py\", line 11, in <module>\r\n    from vnpy.api.xtp import *\r\n  File \"C:\\ProgramData\\Anaconda2\\lib\\site-packages\\vnpy\\api\\xtp\\__init__.py\", li\r\nne 3, in <module>\r\n    from vnxtpquote import QuoteApi\r\nImportError: No module named vnxtpquote\u0000\u0000\r\n`\r\n\r\n",
    "comments_url": "https://api.github.com/repos/vnpy/vnpy/issues/653/comments",
    "author": "ykrank",
    "comments": [
      {
        "user": "vnpy",
        "created_at": "2017-12-22T01:11:18Z",
        "body": "\u5728run.py\u91cc\u628a\u52a0\u8f7dxtpGateway\u7684\u4ee3\u7801\u53bb\u6389\u8bd5\u8bd5"
      },
      {
        "user": "ykrank",
        "created_at": "2017-12-22T02:05:11Z",
        "body": "\u53bb\u6389xtpGateway\u53ef\u4ee5\u6b63\u5e38\u6253\u5f00\u4e86"
      },
      {
        "user": "vnpy",
        "created_at": "2017-12-22T13:39:17Z",
        "body": "dev\u7684xtpGateway\u8fd8\u6ca1\u5168\u90e8\u5f04\u597d\uff0c\u540e\u7eed\u4f1a\u7ee7\u7eed\u5f00\u53d1"
      }
    ],
    "satisfaction_conditions": [
      "Explanation of why the 'vnxtpquote' module is missing or unavailable",
      "Guidance on resolving xtpGateway-related dependencies in the dev branch",
      "Workaround for running the example without complete gateway implementations",
      "Clarification about the development status of specific modules"
    ],
    "_classification": {
      "category": "Requires build environment but hard to be dockerized",
      "timestamp": "2025-04-04 23:52:16"
    }
  },
  {
    "number": 597,
    "title": "\u628a\u76c8\u900f\u5b98\u65b9PY3\u63a5\u53e3\u4e0d\u6210\u529f",
    "created_at": "2017-11-11T09:55:59Z",
    "closed_at": "2017-11-15T00:26:36Z",
    "labels": [
      "help wanted"
    ],
    "url": "https://github.com/vnpy/vnpy/issues/597",
    "body": "## \u73af\u5883\r\n\r\n* \u64cd\u4f5c\u7cfb\u7edf: \u5982Windows 7\u6216\u8005Ubuntu 16.04\r\n* Anaconda\u7248\u672c: \u5982Anaconda 5.0.0 Python 3.6  64\u4f4d\r\n* vn.py\u7248\u672c: \u5982v1.7.1\u53d1\u884c\u7248\u6216\u8005dev branch 20170920\uff08\u4e0b\u8f7d\u65e5\u671f\uff09  \u6700\u65b0\u7248\u7684DEV \r\n\r\n## Issue\u7c7b\u578b\r\n\u4e09\u9009\u4e00\uff1aBug/Enhancement/Question\r\n\r\n## \u9884\u671f\u7a0b\u5e8f\u884c\u4e3a\r\n\r\n\r\n## \u5b9e\u9645\u7a0b\u5e8f\u884c\u4e3a\r\n\r\n\r\n## \u91cd\u73b0\u6b65\u9aa4\r\n\r\n\u6211\u628a\u5b98\u65b9\u7684\u793a\u4f8b\u590d\u5236\u8fdb\u53bb\u91cc\u9762\u7684ibWrapper\r\n\r\nclass IbWrapper(TestWrapper,TestClient):\r\n    \"\"\"IB\u56de\u8c03\u63a5\u53e3\u7684\u5b9e\u73b0\"\"\"\r\n\r\n    #----------------------------------------------------------------------\r\n    def __init__(self, gateway):\r\n        \"\"\"Constructor\"\"\"\r\n        #super(IbWrapper, self).__init__()\r\n        TestWrapper.__init__(self)\r\n        TestClient.__init__(self, wrapper=self)\r\n\r\n\r\n\r\n\u9488\u5bf9Bug\u7c7b\u578bIssue\uff0c\u8bf7\u63d0\u4f9b\u5177\u4f53\u91cd\u73b0\u6b65\u9aa4\u4ee5\u53ca\u62a5\u9519\u622a\u56fe\r\n\u7ed3\u679c\u4e00\u76f4\u8fd9\u6837\uff0c\u6b7b\u5faa\u73af\u4e86\u5417\uff1f\r\n\r\n2017-11-11 17:52:12,187  DEBUG: acquiring lock\r\n2017-11-11 17:52:12,188  DEBUG: acquired lock\r\n2017-11-11 17:52:13,189  DEBUG: exception from recvMsg (<class 'socket.timeout'>, timeout('timed out',), <traceback object at 0x000001498ECB3708>)\r\n2017-11-11 17:52:13,190  DEBUG: releasing lock\r\n2017-11-11 17:52:13,191  DEBUG: release lock\r\n2017-11-11 17:52:13,192  DEBUG: reader loop, prevBuf.size: 0 recvd size: 0 buf b''\r\n2017-11-11 17:52:13,193  DEBUG: acquiring lock\r\n2017-11-11 17:52:13,194  DEBUG: acquired lock\r\n",
    "comments_url": "https://api.github.com/repos/vnpy/vnpy/issues/597/comments",
    "author": "viponedream",
    "comments": [
      {
        "user": "vnpy",
        "created_at": "2017-11-13T01:26:26Z",
        "body": "\u611f\u89c9\u7a0b\u5e8f\u4e00\u76f4\u5728\u5c1d\u8bd5\u83b7\u53d6\u4e00\u628a\u9501\uff08\u53ef\u80fd\u662f\u8bfb\u53d6socket\u6570\u636e\u7684\uff09\uff0c\u7136\u540e\u4e0d\u505c\u7684\u5931\u8d25\u91cd\u590d\n\n---\n\n\u6211\u6ca1\u770b\u8fc7ib\u5b98\u65b9PY API\u7684\u6e90\u4ee3\u7801\uff0c\u8fd9\u4e2a\u5f97\u4e00\u6b65\u6b65debug\u6765\u67e5\u4e86"
      },
      {
        "user": "viponedream",
        "created_at": "2017-11-13T06:35:35Z",
        "body": "\u8c22\u8c22\uff0c\u5b98\u65b9\u7684\u7248\u672c\u641e\u4e0d\u6765\u3002\r\n\u8fd8\u662f IBPY\u7684\u90a3\u4e2a\u7248\u672c\u7b80\u5355\u660e\u4e86\u3002\r\n\u4ee5\u524d\u4f60\u7528\u8fc7\u90a3\u4e2a IBPY\u7684\uff0c\u4e0d\u77e5\u9053\u8be5\u7248\u672c\u53ef\u6709\u5b58\uff1f\u56e0\u4e3a IBPY\u662f\u517c\u5bb9py2, py3\u7684\u3002\u6709\u7684\u8bdd\u5c31\u65b9\u4fbf\u7701\u4e8b\u4e86\u3002\u76f4\u63a5\u53ef\u4ee5\u7528\u5728PY3\u4e0a\u4e86\u3002"
      },
      {
        "user": "vnpy",
        "created_at": "2017-11-14T01:35:00Z",
        "body": "\u53ef\u4ee5\u770bGithub\u7684\u5386\u53f2\uff0c\u5e94\u8be5\u80fd\u627e\u5230"
      },
      {
        "user": "viponedream",
        "created_at": "2017-11-14T22:46:20Z",
        "body": "\u8c22\u8c22\uff0c\u5df2\u7ecf\u5bf9\u63a5\u4e0a\u4e86\u3002\u8fd8\u662f\u4f60\u7684\u6846\u67b6\u517c\u5bb9\u6027\u5f3a\u3002\u51e0\u4e4e\u4e0d\u7528\u600e\u4e48\u6539\uff0c\u76f4\u63a5\u628a\u5b98\u65b9\u7684\u63a5\u53e3\u5957\u4e0a\u5373\u53ef\u7528\u3002"
      },
      {
        "user": "vnpy",
        "created_at": "2017-11-15T00:26:36Z",
        "body": "\u54c8\u54c8\uff0cOK"
      }
    ],
    "satisfaction_conditions": [
      "Solution must demonstrate Python 3 compatibility for IB API integration",
      "Requires minimal code modification to existing vn.py framework",
      "Maintain compatibility with vn.py's existing architecture",
      "Clear documentation of version compatibility requirements"
    ],
    "_classification": {
      "category": "Requires build environment but hard to be dockerized",
      "timestamp": "2025-04-04 23:52:24"
    }
  },
  {
    "number": 382,
    "title": "\u73af\u5883\u642d\u5efa\u95ee\u9898",
    "created_at": "2017-06-30T17:00:19Z",
    "closed_at": "2017-07-05T01:28:20Z",
    "labels": [
      "help wanted"
    ],
    "url": "https://github.com/vnpy/vnpy/issues/382",
    "body": "\u53c2\u8003\u6587\u6863\u4e0a\u7684\u8981\u6c42\u8fdb\u884c\u73af\u5883\u5b89\u88c5\uff0c\u76ee\u524d\u6709\u4e24\u4e2a\u95ee\u9898\uff0c\u65b9\u4fbf\u8bf7\u5e2e\u5fd9\u770b\u4e0b\r\n1.\u5b89\u88c5  conda install -c quantopian ta-lib=0.4.9\r\n\u62a5\u9519\u5982\u4e0b\uff1a\r\n```\r\nUnsatisfiableError: The following specifications were found to be in conflict:\r\n  - matplotlib\r\n  - navigator-updater\r\n  - ta-lib 0.4.9*\r\nUse \"conda info <package>\" to see the dependencies for each package.\r\n```\r\n\r\n2. from vnctpmd import MdApi\r\nImportError: DLL load failed: \u627e\u4e0d\u5230\u6307\u5b9a\u7684\u6a21\u5757\u3002",
    "comments_url": "https://api.github.com/repos/vnpy/vnpy/issues/382/comments",
    "author": "wudixy",
    "comments": [
      {
        "user": "vnpy",
        "created_at": "2017-07-03T01:06:16Z",
        "body": "1. \u8bf7\u95ee\u4f60\u7684Anaconda\u7248\u672c\u662f\uff1f\u5efa\u8bae\u75284.0.0\r\n2. vcredist x86\u6ca1\u88c5"
      },
      {
        "user": "qk-huang",
        "created_at": "2017-07-04T11:19:40Z",
        "body": "\u540c\u6837\u95ee\u9898\n\n---\n\n\u4f7f\u7528\u65b9\u6cd5\u4e00\u4e00\u952e\u5b89\u88c5\u89e3\u51b3\u4e86\n\n---\n\n\u597d\u5427\uff0c\u5c45\u7136anaconda 4.00\u5b8c\u7f8e\u89e3\u51b3"
      },
      {
        "user": "wudixy",
        "created_at": "2017-07-04T16:17:31Z",
        "body": "\u641e\u5b9a\u4e86\uff0c\u4e0d\u719f\u6089anaconda \u865a\u62df\u73af\u5883\u9020\u6210\u3002\u4f46\u73b0\u5728\u8fde\u63a5\u65f6\uff0c\u603b\u662f\u8bf4\uff0cCTP\uff0c\u4e0d\u5408\u6cd5\u7684\u767b\u5f55\uff0c\u9519\u8bef\u4ee3\u7801\uff1a3,\u6000\u7591\u662f\u7528\u6237\u540d\u5bc6\u7801\u9519\u8bef\uff0c\u4f46\u8bd5\u9a8c\u4e86\u597d\u51e0\u6b21\uff0c\u751a\u81f3\u6362\u4e86\u8d26\u53f7 \uff0c\u90fd\u4e0d\u5bf9\uff0c\u914d\u7f6e\u6587\u4ef6\u5e94\u8be5\u662f\u5bf9\u7684\uff0c\u6211\u968f\u4fbf\u6362\u4e86IP\uff0c\u9a6c\u4e0a\u5c31\u8fde\u4e0d\u4e0a\u4e86\u3002\r\n{\r\n    \"brokerID\": \"9999\", \r\n    \"mdAddress\": \"tcp://180.168.146.187:10031\", \r\n    \"tdAddress\": \"tcp://180.168.146.187:10030\", \r\n    \"userID\": \"mmmm\",\r\n    \"password\": \"password\"\r\n}\r\n\r\n\u4e0a\u9762\u8fd9\u4e2aCTP\uff0c\u4e0d\u5408\u6cd5\u767b\u5f55\uff0c\u4e5f\u641e\u5b9a\u4e86\uff0c\u8981\u7528investorId"
      },
      {
        "user": "vnpy",
        "created_at": "2017-07-05T01:28:20Z",
        "body": "\u95ee\u9898\u89e3\u51b3\uff0c\u5173\u95edissue"
      }
    ],
    "satisfaction_conditions": [
      "Resolve package version conflicts in Anaconda environment",
      "Ensure required runtime dependencies (like Visual C++ Redistributable) are installed",
      "Provide correct CTP connection parameters including authentication details",
      "Clear guidance on environment isolation practices"
    ],
    "_classification": {
      "category": "Requires build environment but hard to be dockerized",
      "timestamp": "2025-04-04 23:52:54"
    }
  },
  {
    "number": 336,
    "title": "ubuntu \u7f16\u8bd1lts\u5931\u8d25",
    "created_at": "2017-05-28T06:37:58Z",
    "closed_at": "2017-06-02T14:37:08Z",
    "labels": [
      "help wanted"
    ],
    "url": "https://github.com/vnpy/vnpy/issues/336",
    "body": "\u8fdb\u5165 vnpy/vn.api/vn.lts\r\n\u8fd0\u884c\u4e00\u4e0b\u6307\u4ee4\uff1a\r\nmkdir build\r\ncd build\r\ncmake ..\r\nmake VERBOSE=1 -j 1\r\n\u62a5\u4e0b\u9762\u9519\u8bef\r\n/usr/bin/cmake -H/work/finance/vn/vnpy/vn.api/vn.lts -B/work/finance/vn/vnpy/vn.api/vn.lts/build --check-build-system CMakeFiles/Makefile.cmake 0\r\n/usr/bin/cmake -E cmake_progress_start /work/finance/vn/vnpy/vn.api/vn.lts/build/CMakeFiles /work/finance/vn/vnpy/vn.api/vn.lts/build/CMakeFiles/progress.marks\r\nmake -f CMakeFiles/Makefile2 all\r\nmake[1]: Entering directory '/work/finance/vn/vnpy/vn.api/vn.lts/build'\r\nmake -f CMakeFiles/vnltsmd.dir/build.make CMakeFiles/vnltsmd.dir/depend\r\nmake[2]: Entering directory '/work/finance/vn/vnpy/vn.api/vn.lts/build'\r\ncd /work/finance/vn/vnpy/vn.api/vn.lts/build && /usr/bin/cmake -E cmake_depends \"Unix Makefiles\" /work/finance/vn/vnpy/vn.api/vn.lts /work/finance/vn/vnpy/vn.api/vn.lts /work/finance/vn/vnpy/vn.api/vn.lts/build /work/finance/vn/vnpy/vn.api/vn.lts/build /work/finance/vn/vnpy/vn.api/vn.lts/build/CMakeFiles/vnltsmd.dir/DependInfo.cmake --color=\r\nDependee \"/work/finance/vn/vnpy/vn.api/vn.lts/build/CMakeFiles/vnltsmd.dir/DependInfo.cmake\" is newer than depender \"/work/finance/vn/vnpy/vn.api/vn.lts/build/CMakeFiles/vnltsmd.dir/depend.internal\".\r\nDependee \"/work/finance/vn/vnpy/vn.api/vn.lts/build/CMakeFiles/CMakeDirectoryInformation.cmake\" is newer than depender \"/work/finance/vn/vnpy/vn.api/vn.lts/build/CMakeFiles/vnltsmd.dir/depend.internal\".\r\nScanning dependencies of target vnltsmd\r\nmake[2]: Leaving directory '/work/finance/vn/vnpy/vn.api/vn.lts/build'\r\nmake -f CMakeFiles/vnltsmd.dir/build.make CMakeFiles/vnltsmd.dir/build\r\nmake[2]: Entering directory '/work/finance/vn/vnpy/vn.api/vn.lts/build'\r\nmake[2]: Warning: File 'CMakeFiles/vnltsmd.dir/depend.make' has modification time 0.69 s in the future\r\n[ 16%] Building CXX object CMakeFiles/vnltsmd.dir/vnltsmd/vnltsmd/vnltsmd.cpp.o\r\n/usr/bin/c++   -DBUILD_LTS_MD -DBUILD_LTS_QRY -DBUILD_LTS_TD -DUSE_64BITS -Dvnltsmd_EXPORTS -I/work/finance/vn/vnpy/vn.api/vn.lts/ltsapi -I/work/finance/vn/vnpy/vn.api/vn.lts/LTS_MD_PATH -I/work/finance/vn/vnpy/vn.api/vn.lts/LTS_TD_PATH -I/work/finance/vn/vnpy/vn.api/vn.lts/LTS_QRY_PATH -I/usr/include/python2.7  -fPIC -std=c++11 -O3 -DNDEBUG -fPIC   -o CMakeFiles/vnltsmd.dir/vnltsmd/vnltsmd/vnltsmd.cpp.o -c /work/finance/vn/vnpy/vn.api/vn.lts/vnltsmd/vnltsmd/vnltsmd.cpp\r\n[ 33%] Linking CXX shared library lib/vnltsmd.so\r\n/usr/bin/cmake -E cmake_link_script CMakeFiles/vnltsmd.dir/link.txt --verbose=1\r\n/usr/bin/c++  -fPIC  -fPIC -std=c++11 -O3 -DNDEBUG  -shared -Wl,-soname,vnltsmd.so -o lib/vnltsmd.so CMakeFiles/vnltsmd.dir/vnltsmd/vnltsmd/vnltsmd.cpp.o  -L/work/finance/vn/vnpy/vn.api/vn.lts/ltsapi -lboost_python -lboost_thread -lboost_date_time -lboost_system -lboost_chrono -lboost_atomic -lpthread -lpython2.7 -lsecuritymduserapi -Wl,-rpath,/work/finance/vn/vnpy/vn.api/vn.lts/ltsapi\r\n/usr/bin/ld:/work/finance/vn/vnpy/vn.api/vn.lts/ltsapi/libsecuritymduserapi.so: file format not recognized; treating as linker script\r\n/usr/bin/ld:/work/finance/vn/vnpy/vn.api/vn.lts/ltsapi/libsecuritymduserapi.so:1: syntax error\r\ncollect2: error: ld returned 1 exit status\r\nCMakeFiles/vnltsmd.dir/build.make:103: recipe for target 'lib/vnltsmd.so' failed\r\nmake[2]: *** [lib/vnltsmd.so] Error 1\r\nmake[2]: Leaving directory '/work/finance/vn/vnpy/vn.api/vn.lts/build'\r\nCMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/vnltsmd.dir/all' failed\r\nmake[1]: *** [CMakeFiles/vnltsmd.dir/all] Error 2\r\nmake[1]: Leaving directory '/work/finance/vn/vnpy/vn.api/vn.lts/build'\r\nMakefile:83: recipe for target 'all' failed\r\nmake: *** [all] Error 2\r\n\r\n\u770b\u8d77\u6765\u662f\u7f16\u8bd1\u94fe\u63a5\u7684\u9519\u8bef,\u600e\u4e48\u89e3\u51b3\r\n",
    "comments_url": "https://api.github.com/repos/vnpy/vnpy/issues/336/comments",
    "author": "zsluedem",
    "comments": [
      {
        "user": "vnpy",
        "created_at": "2017-05-30T09:11:29Z",
        "body": "\u8bd5\u8bd5\u76f4\u63a5\u7528bash build.sh\u80fd\u5426\u7f16\u8bd1\uff1f"
      },
      {
        "user": "zsluedem",
        "created_at": "2017-05-31T00:40:58Z",
        "body": "\u4e0d\u53ef\u4ee5\u554a\uff5e\uff01\u62a5\u540c\u6837\u7684\u9519\uff5e\uff01"
      },
      {
        "user": "vnpy",
        "created_at": "2017-06-02T14:37:20Z",
        "body": "\u6211\u8fd9\u8fb9\u6d4b\u8bd5\u4e86\u4e0b\uff0c\u53d1\u73b0\u4e86\u540c\u6837\u7684\u95ee\u9898\uff0c\u5e94\u8be5\u662f\u4e0a\u6b21\u66f4\u65b0\u6709\u4e1c\u897f\u51fa\u9519\u4e86\uff0c\u4f1a\u5c3d\u5feb\u4fee\u590d\n\n---\n\n\u627e\u5230\u95ee\u9898\u6240\u5728\u4e86\uff1av1.6.1\u7684ltsapi\u76ee\u5f55\u4e0b\u7684\u4e09\u4e2alib\u5f00\u5934\u7684.so\u6587\u4ef6\uff0c\u53ef\u80fd\u56e0\u4e3a\u4e4b\u524d\u540c\u6b65\u7684\u539f\u56e0\u5f04\u574f\u4e86\uff0c\u76ee\u524d\u90fd\u662f\u9519\u8bef\u7684\u3002\r\n\r\n\u5220\u9664\u4e09\u4e2alib\u5f00\u5934\u7684.so\u6587\u4ef6\uff0c\u7136\u540e\u5c06\u4e09\u4e2asecurity\u5f00\u5934\u7684.so\u6587\u4ef6\u52a0\u4e0alib\u524d\u7f00\uff08\u8fd9\u4e09\u4e2a\u6587\u4ef6\u662f\u6b63\u5e38\u7684\uff09\uff0c\u518d\u6267\u884c\u7f16\u8bd1\u5373\u53ef\u3002\n\n---\n\n\u4e5f\u5c06\u66f4\u65b0\u63a8\u9001\u5230\u4e86dev\u4e2d"
      }
    ],
    "satisfaction_conditions": [
      "Identify and resolve issues with corrupted/invalid shared library (.so) files",
      "Ensure proper naming conventions for required library files",
      "Provide a method to validate library file integrity",
      "Maintain compatibility with existing build system requirements",
      "Address file synchronization-related corruption risks"
    ],
    "_classification": {
      "category": "Requires build environment but hard to be dockerized",
      "timestamp": "2025-04-04 23:53:07"
    }
  }
]